rllib/agents/pg/README.md

```diff
@@ -0,0 +1,13 @@
+Policy Gradient (PG)
+====================
+
+An implementation of a vanilla policy gradient algorithm for TensorFlow and PyTorch. 
+
+**[Detailed Documentation](https://docs.ray.io/en/latest/rllib-algorithms.html#pg)**
+
+Papers
+------
+
+[1] [Policy Gradient Methods for Reinforcement Learning with Function Approximation](https://papers.nips.cc/paper/1713-policy-gradient-methods-for-reinforcement-learning-with-function-approximation.pdf)
```

Comment:
Please remove this section (duplicates docs).

Reply:
done